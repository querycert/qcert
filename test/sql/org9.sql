select *
from employees
except
select *
from employees
where age = 32
;
