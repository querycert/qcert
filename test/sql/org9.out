[{"age": {"nat":34}, "company": 101, "eid": 3, "name": "Jim Does"}, {"age": {"nat":34}, "company": 103, "eid": 5, "name": "Joan Doe"}, {"age": {"nat":35}, "company": 104, "eid": 6, "name": "James Do"}]